src/panoramic/cli/local/writer.py

```diff
@@ -135,3 +136,12 @@ def delete_field(self, field: PanoField):
 
         logger.debug(f'About to delete field {field.id}')
         delete_file(path)
+
+    def write_transform(self, transform: PanoTransform, path: Optional[Path] = None):
```

Comment:
Not sure I like `path` being here - the idea with `FileWriter` is that it abstracts the idea of where the models/fields/transforms etc. are written

Reply:
Dropped in 7abcc31aae5ffda9c7c490dc5275470c5b1d0940